.github/workflows/frontend.yml

```diff
@@ -38,6 +38,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-java@v3
       with:
+        distribution: 'temurin'
```

Comment:
@brianzelip Is this the case of us both doing the same thing independently? It looks like when this is merged we will have double `distribution:` values: yours in quotes, mine without. (No idea which is better or if it matters).

Reply:
ah, I understand now.